### PR TITLE
Lock10 migration guide updates

### PR DESCRIPTION
**Pull request by jeffreylees**
Also interlinking with other docs to provide easier access to details
about new options/events/etc.

### CODE REVIEW

**Comment by auth0build** [comment]
Can one of the admins verify this patch?

File: articles/libraries/lock/v10/migration-guide.md [hunk 1]
```diff
@@ -4,39 +4,74 @@ Heads up! This document is using the latest version of Lock (version 10). See ch
 
 # Lock 9 to Lock 10 Migration Guide
 
-The following instructions assume you are migrating from Lock 9 to the latest Lock 10. If you are upgrading from a preview release, please refer to the [preview changes](#preview-changes). If you came here looking for the migration guide from the "Auth0Widget" to Lock v9, here's the [legacy-migration-guide](/libraries/lock/v9/migration-guide). Otherwise, read on!
+The following instructions assume you are migrating from **Lock 9** to the latest **Lock 10**. If you are upgrading from a preview release of Lock 10, please refer to the [preview changes](#preview-changes). Otherwise, read on!
```
Comment (mpaktiti) [inline articles/libraries/lock/v10/migration-guide.md]: @jeffreylees The [URL](https://auth0-docs-content-pr-2008.herokuapp.com/docs/libraries/lock/v10/migration-guide#preview-changes) does not work. Maybe you meant `#upgrading-from-preview-releases`?

File: articles/libraries/lock/v10/migration-guide.md [hunk 2]
```diff
@@ -4,39 +4,74 @@ Heads up! This document is using the latest version of Lock (version 10). See ch
...
 - You no longer need to call `parseHash` when implementing Redirect Mode. The data returned by that method is provided to the `authenticated` event listener.
+
+### Customizing Options
+- The `show` method no longer takes any arguments. You pass the options to the constructor and you listen for an `authenticated` event instead of providing a callback. You can listen for this event with the [on](/libraries/lock/v10/api#on-event-callback-) method.
+
+### Authenticated Event
+- The `authenticated` event listener has a single argument, an `authResult` object. This object contains the following properties: `idToken`, `accessToken`, `state`, `refreshToken` and `idTokenPayload`. Most of them correspond to the arguments passed to the `show` method's callback.
+
+### Internationalization
 - Not all languages supported by Lock v9 are supported by Lock v10.
```
Comment (mpaktiti) [inline articles/libraries/lock/v10/migration-guide.md]: @jeffreylees Shouldn't we list the languages not supported or redirect to a doc that has this info?

File: articles/libraries/lock/v10/migration-guide.md [hunk 3]
```diff
@@ -4,39 +4,74 @@ Heads up! This document is using the latest version of Lock (version 10). See ch
...
+## Further Reading
+  - Some other options were added, see [New Features page](/libraries/lock/v10/new-features) for details.
+  - Check out the [customization page](/libraries/lock/v10/customization) for more details on all of the customization options that are available.
+  - Take a look at the [api page](/libraries/lock/v10/api) for more details on Lock 10's API.
 
-## Upgrading Between (or From) Preview Releases
+## Upgrading From Preview Releases
 
-This is a summary of what you absolutely need to know before upgrading between preview releases. For the full list of changes, please see the project's [CHANGELOG](https://github.com/auth0/lock/blob/v10/CHANGELOG.md).
+**This section is only pertinent if you are currently using a pre-release version of Lock 10, and wish to update to the release version.** It is a summary of what you absolutely need to know before upgrading between preview releases. For the full list of changes, please see the project's [CHANGELOG](https://github.com/auth0/lock/blob/v10/CHANGELOG.md).
```
Comment (mpaktiti) [inline articles/libraries/lock/v10/migration-guide.md]: @jeffreylees The [CHANGELOG](https://github.com/auth0/lock/blob/v10/CHANGELOG.md) URL is broken.

**Comment by jeffreylees** [comment]
@mpaktiti changes made

**Pull request merged**